### ISSUE TITLE
Hardware SPI and micropython REPL

### ISSUE THREAD

**Issue by hstarmans** [atomic14/esp32-s3-pinouts#25]
**hardware spi**
Some sources state that hardware SPI is only available on the [following](https://www.luisllamas.es/en/esp32-s3-hardware-details-pinout)
SPI | MOSI | MISO | CLK | CS 
-- | -- | -- | -- | -- 
SPI2 | GPIO 35 | GPIO 37 | GPIO 36 | GPIO 39 
SPI3 | GPIO 11 | GPIO 13 | GPIO 12 | GPIO 10  

**Micropython REPL**

If you are using micropython avoid GPIO 1 and 3.
These are connected to repl. You can't use these while connected to Micropython via USB, [source](https://docs.micropython.org/en/latest/esp8266/tutorial/repl.html)

**Comment by eddiegb** [comment]
Yes but I've seen more than one source (e.g. https://randomnerdtutorials.com/esp32-s3-devkitc-pinout-guide/ ) stating that GPIOs 10-13 correspond to SPI2 (commonly known as HSPI) and GPIOs 35-37,39 correspond to SPI3 (commonly known as VSPI). The source you linked to has them the other way around.

Another source ( https://curiousscientist.tech/blog/esp32-s3-multiple-spi-buses-spi-fspi-hspi ) says that ESP32-S3 models with PSRAM (e.g. "N8R8" variants) cannot use GPIO 35, GPIO 36 and GPIO 37 because they are used for internal communication between the ESP32-S3 chip and the PSRAM. This source also says that there is no such thing as "VSPI" in the S3, there is only "FSPI" which corresponds to SPI2 (GPIOs 10-13) and "HSPI"  which corresponds to SPI3 (GPIOs 35-37,39).
